plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties

```diff
@@ -0,0 +1,347 @@
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreAggregate.outputType.name=Type en sortie
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreAggregate.outputType.description=Types de donn\u00e9es en sortie
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreAggregate.objectId.name=ID de l'objet
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreAggregate.function.name=Fonction
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreAggregate.description.name=Commentaire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreForeignDataWrapper.handler.name=Gestionnaire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreForeignDataWrapper.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreForeignDataWrapper.owner.name=Propri\u00e9taire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreForeignServer.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreForeignServer.owner.name=Propri\u00e9taire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreUserMapping.foreignServer.name=Serveur distant
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreUserMapping.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreUserMapping.serverOptions.name=Options du serveur
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.accessMethod.name=M\u00e9thode d'acc\u00e8s
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.accessMethod.description=M\u00e9thode d'acc\u00e8s
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.clustered.name=Regroup\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.description.name=Commentaires
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.indexRelSize.description=Espace disque utilis\u00e9 par l'index
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.immediate.name=Imm\u00e9diat
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.primary.name=Primaire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.primary.description=Cet index repr\u00e9sente la cl\u00e9 primaire de la table (dans ce cas, le champ unique doit \u00e9galement \u00eatre vrai)
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.ready.name=Pr\u00eat
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.ready.description=Si true, l?index est pr\u00eat \u00e0 ajouter des donn\u00e9es. Une valeur false indique que l?index est ignor\u00e9 pour INSERTS/UPDATES.
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.unique.description=Index unique
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.name=Valide
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.description=Si true, l?index peut \u00eatre utilis\u00e9 dans les requ\u00eates. False signifie que l?index est peut-\u00eatre incomplet : il sera cependant modifi\u00e9 par les commandes INSERT/UPDATE, mais il est s\u00fbr de l?utiliser dans des requ\u00eates impossibles. Si elle est unique, la propri\u00e9t\u00e9 d?unicit\u00e9 n?est pas non plus garantie.
```

Comment:
`l?index`, And I see more issues with the apostrophe symbol. Could you please fix it?

Reply:
Fixed